webpack/.stylelintrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
there are 100s of rules are [available](http://stylelint.io/user-guide/rules/).

I will add them in separate commit with fixes around.